frontend/src/app/services/query-sender.service.ts

```diff
@@ -57,5 +58,16 @@ export class QuerySenderService {
       .catch(error => Observable.throw(error))
   }
 
+  /**
+   * Retrieve authorID in localStorage or create it if not existing
```

Comment:
Facultative: you should add some information about how the ID is generated (I mean: 'id are generated randomly.')